.github/workflows/checks.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     env:
-      DEVNET_SHA: c6ffb99597eeea58d1a6a30fc061de69e586c779
+      DEVNET_SHA: 47ee2a73c227ee356f344ce94e5f61871299be80
```

Comment:
In the future please restrain from introducing such changes in unrelated PRs - we should bump devnet in separate one (that can be stacked with this or merged independently)

Reply:
I think it's related, we're just bumping devnet to the version that introduced goerli -> sepolia change.